project.clj

```diff
@@ -34,7 +34,7 @@
 
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [cheshire "5.5.0"]
-                 [org.clojure/core.match "0.2.2"]
+                 [org.clojure/core.match "0.3.0-alpha4" :exclusions [org.clojure/tools.analyzer.jvm]]
```

Comment:
We would have to do this for upgrading to latest tk etc. anyway, so I agree with what @mullr has done here. Annoying though :-).